server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyToResponseBody.scala

```diff
@@ -42,44 +39,31 @@ class NettyToResponseBody extends ToResponseBody[NettyResponse, NoStreams] {
         val byteBuffer = v.asInstanceOf[ByteBuffer]
         (ctx: ChannelHandlerContext) => ByteBufNettyResponseContent(ctx.newPromise(), Unpooled.wrappedBuffer(byteBuffer))
```

Comment:
we don't need the reactive publisher in this case?

Reply:
so only the chunked responses are a problem?

Reply:
Yes, exactly. Only stream, file, etc.